doc/user_guide/index.rst

```diff
@@ -7,17 +7,22 @@ Contents:
 * `Introduction <Introduction.html>`_
    What is ParamBokeh, and how to use it?
 
-* `Usage in Bokeh Apps <Bokeh_App.html>`_
-   How to use ParamBokeh in a Bokeh application.
-   
 * `ViewParameters <View_Parameters.html>`_
    Dynamically control some visual output.
 
+* `Usage in Bokeh Apps <Bokeh_App.html>`_
+   How to use ParamBokeh in a Bokeh server application.
```

Comment:
Not sure "Usage in" adds anything here or below.

Reply:
Agreed